content/swift/concepts/loops/loops.md

```diff
@@ -1,6 +1,6 @@
 ---
 Title: 'Loops'
-Description: 'The for-in loop is used to iterate over collections, including strings and ranges. swift for char in "hehe" { print(char) }  The output would be:'
+Description: 'The for-in loop is used to iterate over collections, including strings and ranges.'
```

Comment:
"for - in loop should be bolded below.

Reply:
@SSwiniarski I'm opting for a general loops description here